samples/client/petstore/python-experimental/petstore_api/api/another_fake_api.py

```diff
@@ -102,7 +102,7 @@ def __call_123_test_special_tags(self, body, **kwargs):  # noqa: E501
                 '_check_return_type', True
             )
             kwargs['_host_index'] = kwargs.get('_host_index', 0)
-            kwargs['body'] = body
+            kwargs['body'] = body   # noqa: E501
```

Comment:
Can you regenerate the samples?
This file was generate using an outdated version of the template.

Reply:
done.